features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -2,6 +2,7 @@
   "authentication.consent.prompt": true,
   "authentication.consent.subject.prompt": true,
   "authentication.map_federated_users_to_local": false,
+  "authentication.enable_scope_based_claim_filtering": true,
```

Comment:
is this also added to the migration guide? since you are changing the default behaviour, this should be mentioned properly